### PR TITLE
python3Packages.gdown: fix build

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

Failed to build because of missing six dependency. I assume it was previously propagated from some other dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Off topic question: is this tool faster than downloading via the web?

**Comment by mweinelt** [comment]
I don't use it, it is just a dependency somewhere downstream of umap-learn.

File: pkgs/development/python-modules/gdown/default.nix [hunk 1]
```diff
@@ -16,7 +16,12 @@ buildPythonApplication rec {
     sha256 = "4b3a1301e57bfd8dce939bf25ef8fbb4b23967fd0f878eede328bdcc41386bac";
   };
 
-  propagatedBuildInputs = [ filelock requests tqdm setuptools ];
+  propagatedBuildInputs = [
+    filelock
+    requests
+    six
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/gdown/default.nix:22]:
```suggestion
    requests
    setuptools
    six
```

```
results/gdown/bin/gdown --version
Traceback (most recent call last):
  File "/nix/store/x8m83myhw1nsa5i7kj3h64xn4ds4l8z7-gdown-3.12.2/bin/.gdown-wrapped", line 6, in <module>
    from gdown.cli import main
  File "/nix/store/x8m83myhw1nsa5i7kj3h64xn4ds4l8z7-gdown-3.12.2/lib/python3.8/site-packages/gdown/__init__.py", line 3, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

**Comment by SuperSandro2000** [comment]
Already fixed by https://github.com/NixOS/nixpkgs/commit/a4c701e7e3241d1949ba1e802cb2dd6c87a5ec32

**Comment by jonringer** [comment]
oh, sorry

**Pull request closed without merging**